mlir/include/mlir/Dialect/MIOpen/MIOpenOps.td

```diff
@@ -296,19 +296,28 @@ def MIOpen_MFMAV2Op:
 // xdlops_gemm_V2
 def MIOpen_XdlopsGemmV2Op:
     MIOpen_Op<"xdlops_gemm_v2">,
-    Arguments<(ins MemRefOf<[F32, F16, BF16]>:$matrixA,
-                   MemRefOf<[F32, F16, BF16]>:$matrixB,
+    Arguments<(ins MemRefOf<[F32, F16, I16]>:$matrixA,
+                   MemRefOf<[F32, F16, I16]>:$matrixB,
                    Index:$threadOffsetA,
                    Index:$threadOffsetB,
-                   MemRefOf<[F32, F16, BF16]>:$bufferA,
-                   MemRefOf<[F32, F16, BF16]>:$bufferB,
-                   Variadic<VectorOfRankAndType<[1], [F32, F16, BF16]>>:$vectorCs)>,
-    Results<(outs Variadic<VectorOfRankAndType<[1], [F32, F16, BF16]>>: $vectorDs)> {
+                   MemRefOf<[F32, F16, I16]>:$bufferA,
+                   MemRefOf<[F32, F16, I16]>:$bufferB,
+                   Variadic<VectorOfRankAndType<[1], [F32, F16]>>:$vectorCs)>,
+    Results<(outs Variadic<VectorOfRankAndType<[1], [F32, F16]>>: $vectorDs)> {
   let summary = "XDLOPS GEMM V2";
   let description = [{
     The `miopen.xdlops_gemm_v2` op is an abstraction of doing GEMM based on XDLOPS.
     It would employ a series of `miopen.mfma` operations.
   }];
 }
 
+def MIOpen_DataConvertOp :
+    MIOpen_Op<"data_convert">,
+    Arguments<(ins F32:$in)>,
+    Results<(outs I16:$out)> {
+  let summary = "from data type to another";
+  let description = [{
+    The `miopen.data_convert` op transforms some data type.
```

Comment:
Please state more clearly what this op does actually.

Reply:
already changed